common/src/main/java/com/aliyun/emr/rss/common/haclient/RssHARetryClient.java

```diff
@@ -95,13 +96,64 @@ public static String encodeRequestId(String uuid, long callId) {
   /**
    * For message sent by Master itself like ApplicationLost or WorkerLost, we should set requestId
    * manually.
-   *
-   * @return
    */
   public static String genRequestId() {
     return encodeRequestId(UUID.randomUUID().toString(), nextCallId());
   }
 
+  public void send(TransportMessage req) throws Throwable {
+    // Send a one-way message. Because we need to know whether the leader between Masters has
+    // switched, we must adopt a synchronous method, but for a one-way message, we don’t care
+    // whether it can be sent successfully, so we adopt an asynchronous method. Therefore, we
+    // choose to use one Thread pool to use synchronization.
+    oneWayMessageSender.submit(
+        () -> {
+          try {
+            sendMessageInner(req, TransportMessage.class);
+          } catch (Throwable e) {
+            LOG.warn("Exception occurs while send one-way message.", e);
+          }
+        });
+    LOG.debug("Send message {}.", req);
+  }
+
+  public TransportMessage askSync(TransportMessage req) throws Throwable {
```

Comment:
I'm a little bit hesitate here, there are two directions:
1) as the code in this PR, we also expose `TransportMessage` application layer. I checked the Spark codebase, it does in this way.
2) keep the transparent transformation between `TransportMessage` and `PbXXX` in serde layer, like before